catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -349,11 +249,21 @@ public CreateResponse create(CreateStorageRequest streamCreateRequest)
 
             createResponse = create(createRequest);
         } catch (Exception e) {
```

Comment:
Should we `catch RuntimeException` instead to make sure that if a new type of checked exception that gets thrown in the future we know this code needs to be updated to handle it?